package.json

```diff
@@ -50,5 +50,8 @@
   "bugs": {
     "url": "https://github.com/amasad/sane/issues"
   },
-  "homepage": "https://github.com/amasad/sane"
+  "homepage": "https://github.com/amasad/sane",
+  "optionalDependencies": {
+    "fsevents": "^1.1.1"
```

Comment:
Yeah agreed. I'm really reluctant to do it. But the problem is badd enough that the NodeWatcher is not usable on some machines (including mine).